src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -39,6 +39,13 @@
 
     private boolean voltageRemoteControl = false;
 
+    public enum LowImpedanceBranchMode {
+        REPLACE_BY_COUPLER,
```

Comment:
I am not sure about these choices. I think that I prefer more explicit and more symetric ones:
- ````REPLACE_BY_ZERO_IMPEDANCE_LINE```` ;
- ````REPLACE_BY_LOW_IMPEDANCE_LINE````.
What do you think ? Is a coupler the right term ?

Reply:
REPLACE_BY_ZERO_IMPEDANCE_LINE
REPLACE_BY_MIN_LOW_IMPEDANCE_LINE
?